docs/register-phase-1-stake.md

```diff
@@ -0,0 +1,154 @@
+WalletPass = "your-wallet-password"  # Replace with your wallet password
+
+[walletrpcconfig]
+Host = "127.0.0.1:19001"  # Bitcoin Wallet RPC endpoint
+User = "your-rpc-user"    # Replace with your Bitcoin RPC username
+Pass = "your-rpc-pass"    # Replace with your Bitcoin RPC password
+
+[chain]
+Network = "signet"  # Bitcoin network (signet for testnet)
+```
+
+Finally, start the `btcstaker` daemon, which will manage your connections to both 
+Babylon and Bitcoin nodes, monitor transactions, and handle the staking process.
+Add the location for the above created `config.json` file to the command:
+
+```shell
+stakerd --configfile <config-file-path> admin start
+```
+
+## Register your stake
+
+After configuring and starting the `btcstaker` daemon, you'll need the 
+`global_parameters.json` file (created during prerequisites) for the next steps.
+
+A few notes about the `global-parameters.json` file:
+1. It is used to retrieve covenant keys and quorum from the parameters version 
+   you staked in.
+2. It can also be used to parse your Bitcoin staking transaction to extract 
+   important information from the `OP_RETURN` data, which contains:
+   - The staking period you chose
+   - The finality provider you chose
+
+> **⚡ Note**: While these parameters can be provided manually, the global 
+> parameters file provides an easier user experience. This guide is only for 
+> valid, confirmed Bitcoin staking transactions with valid `OP_RETURN` data. If 
+> your transaction has malformed `OP_RETURN` data or isn't a confirmed staking 
+> transaction, to check your eligibility please refer to the 
+> [registration eligibility guide](registration-eligibility.md).
+
+Now that you have the `global-parameters.json` file, you can register your 
+Bitcoin stake by running the following command:
+
+```shell
+stakercli daemon stake-from-phase1 <global-parameters-file> \
+    --staking-transaction-hash <your-phase1-tx-hash> \
+    --staker-address <your-btc-address> \
+    --tx-inclusion-height <block-height>
+```
+
+Parameters:
+- `global-parameters-file`: The path to the global parameters file.
+- `your-phase1-tx-hash`: The original hash of your Bitcoin staking transaction.
+- `your-btc-address`: BTC address of the staker in hex.
+- `block-height`: The BTC block height at which your staking transaction was 
+  included.
+
+The above command will output a transaction hash similar to below 
+that you can use to verify the registration on the Babylon blockchain, which 
+you will see in the next section.
+
+```json
+{
+  "babylon_btc_delegation_tx_hash": "<btc-delegation-tx-hash>"
+}
+```
+
+## Verify your registration
+
+Following the registration of your Bitcoin stake, you can verify your 
+registration by checking your balance on the [Babylon Explorer](https://babylon-testnet.l2scan.co) 
+by searching for the `<babylon_btc_delegation_tx_hash>` value.
+
+This will then give you details on your transaction, which includes the state 
```

Comment:
The below states relate to the statuses maintained by `stakerd`. Above you refer to going to the Babylon explorer. Something is missing here I think. In general, I recommend that we check the status of the transaction by stakerd/stakercli commands instead of sending people to a 3rd party explorer.